content/learning-paths/servers-and-cloud-computing/cca-veraison/attestation-token.md

```diff
@@ -0,0 +1,148 @@
+```
+
+Export the installation path and add it to your $PATH environment variable.
+
+```bash
+export PATH=$PATH:/usr/local/go/bin
+```
+Verify the installation by checking that the command outputs the installed version.
+
+```bash
+go version
+```
+
+## Install Git
+
+Verify that `git` is installed using the command below. It should output the version available on your computer.
+
+```bash
+git --version
+```
+
+## Download the Example CCA Attestation Token
+
+Using your preferred web browser, navigate to the [token in the TrustedFirmware-M tools repository](https://github.com/TrustedFirmware-M/tf-m-tools/blob/main/iat-verifier/tests/data/cca_example_token.cbor)
+
+Use GitHub’s download button, located on the right of the upper toolbar, to download the token as a *raw* (binary) file.
+
+![download_raw.png](./download_raw.png)
+
+Place this file in the `$HOME` folder, while keeping the file name the same. The rest of this learning path will use the notation `$HOME/cca_example_token.cbor` as the file path.
+
+{{% notice Note %}}
+You will notice that the filename extension on the example token is `.cbor`, which also denotes the format of the data. CBOR is the Concise Binary Object Representation. You are likely to already be familiar with JSON (the JavaScript Object Notation). JSON provides a standard way to convey nested structures of key-value pairs. CBOR is conceptually the same as JSON. The difference is that CBOR is a binary format, rather than a text-based format like JSON. CBOR is designed for compactness and ease of machine-readability, but at the expense of human-readability. You can learn more about CBOR here.
+{{% /notice %}}
+
+## Build the EVCLI Tool
+
+Now that you have downloaded the example CCA attestation token, the next step is to look inside the token and learn about the data that it contains. Because the token is a binary file, you will need to use a tool to parse the file and display its contents. The tool that you will use is a command-line tool called `evcli` (which is short for the EVidence Command Line Interface – remember that attestation tokens are used to convey evidence about realms and the platforms on which they are hosted).
+
+The `evcli` tool is part of the Veraison open-source project, which was covered in the previous section.
+
+Clone the source code using git as follows:
+
+```bash
+cd $HOME
+git clone https://github.com/veraison/evcli.git
+```
+Change the directory and build the tool:
+
+```bash
+cd evcli
+go build
+```
+
+The tool is quite small, so this should not take long. Once it has built, you can progress to the next step.
+
+## Inspect the CCA Example Attestation Token
+
+Now that you have built the `evcli` command-line tool, you can use it to inspect the contents of the example CCA attestation token that you downloaded earlier.
+
+Run the following command, taking care to substitute the correct path where you stored the CCA example token from the earlier step.
```

Comment:
We can remove the caution about substituting the path because we're using `$HOME` as an anchor.

Reply:
done